modules/oc-mirror-disk-to-mirror-v2.adoc

```diff
@@ -6,24 +6,31 @@
 [id="disk-mirror-v2_{context}"]
 == Mirroring from disk to mirror
 
-You can use the `oc-mirror` plugin v2 to mirror image sets from a disk to a target mirror registry.
+You can use the oc-mirror plugin v2 to mirror image sets from a disk to a target mirror registry.
 
-The `oc-mirror` plugin v2 retrieves container images from a local disk and transfers them to the specified mirror registry.
+The oc-mirror plugin v2 retrieves container images from a local disk and transfers them to the specified mirror registry.
 
 .Procedure
 
-* Process the image set file on the disk and mirror the contents to a target mirror registry by running the following command:
+. Transfer the disk containing mirrored image sets to the environment that contains the target mirror registry.
+
+. Process the image set file on the disk and mirror the contents to a target mirror registry by running the following command:
 +
 [source,terminal]
 ----
-$ oc mirror -c isc.yaml --from file://<file_path> docker://<mirror_registry_url> --v2 <1>
+$ oc mirror -c <image_set_configuration> --from file://<file_path> docker://<mirror_registry_url> --v2
 ----
-<1> Specify the URL or address of the mirror registry where the images are stored and from which they need to be deleted.
++
+where:
+
+<image_set_configuration>:: Specifies the name of the image set configuration file.
+<file_path>:: Specifies the directory on the disk containing the archives. This folder will contain cluster resources generated for you apply to the cluster, for example the ImageDigestMirrorSet (IDMS) or ImageTagMirrorSet (ITMS) resources.
```

Comment:
should we say `This folder will also contain cluster resources generated that can be applied to the cluster` ?

Reply:
If these are present in addition to archives, sure. I honestly can't remember the context here so if that sounds more correct to you, I can make the change

Reply:
yes, IMHO i think that would be good.